dev-proxy-abstractions/IInactivityTimer.cs

```diff
@@ -0,0 +1,8 @@
+namespace DevProxy.Abstractions
+{
+    public interface IInactivityTimer
```

Comment:
I don't think we need this interface, since we don't really use it anywhere and instantiate InactivityTimer ourself in ProxyEngine.